docs/guides/software-lifecycle/continuous-testing/README.md

```diff
@@ -1,3 +1,183 @@
 # Continuous Testing
```

Comment:
Please use the _new_ SLIM Best Practice Guide Template available [here](https://nasa-ammos.github.io/slim/docs/contribute/submit-best-practice/#use-our-standard-guide-template). If there are more sections you wish to include, go for it, but using this standard guide will help with consistency across other SLIM guides.